manage.py

```
@@ -269,9 +269,9 @@ def init():
     if config_name == 'test':
       es.indices.delete(index=app.config['ELASTICSEARCH_SIMILARITY'], ignore=[400, 404])
     es.indices.create(index=app.config['ELASTICSEARCH_SIMILARITY'])
-  except TransportError as e:
+  except (TransportError, RequestError) as e:
     # ignore already existing index
-    if e.error == 'resource_already_exists_exception':
+    if e.error in ['resource_already_exists_exception', 'invalid_index_name_exception']:
       pass
```

as discussed, can you add a warning line here something like "skipped creating index XX because already exists or index name invalid" so we don't silently hide a possible error

---

Sorry, added!